wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Insights > Today's Stats

Fixes #10577 

To test:

---
- On a site that has all four Today's Stats, go to Stats > Insights.
  - Tip: try en.blog.wordpress.com
- Verify all four stats appear:

![all](https://user-images.githubusercontent.com/1816888/49317057-40008f00-f4b0-11e8-939d-85c734310764.png)

---
- On a site that has some Today's Stats, go to Stats > Insights.
  - Tip: try the Field Guide.
- Verify only the stats that have values appear:

![some](https://user-images.githubusercontent.com/1816888/49317111-6d4d3d00-f4b0-11e8-8dfb-ffdd484c92f2.png)

----
- On a site that has no Today's Stats, go to Stats > Insights.
- Verify `No data yet` appears:

![none](https://user-images.githubusercontent.com/1816888/49317169-9f5e9f00-f4b0-11e8-99c7-4b476bb501d7.png)
